src/no/runsafe/nchat/channel/ChannelManager.java

```diff
@@ -49,11 +49,11 @@ public IChatChannel getPrivateChannel(ICommandExecutor player1, ICommandExecutor
 		if (cmp == 0)
 			return null;
 
-		if ((player1 instanceof IPlayer && player2 instanceof IPlayer)
-			&& (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2)
-			|| ignoreHandler.playerIsIgnoring((IPlayer) player2, (IPlayer) player1))
-		)
-			return null;
+		if (player1 instanceof IPlayer && player2 instanceof IPlayer)
+			if (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2)
+				|| ignoreHandler.playerIsIgnoring((IPlayer) player2, (IPlayer) player1)
+			)
+				return null;
```

Comment:
This still feels very messy.. I think we should just add a `eitherPlayerIsIgnoring` and use that.. saves on redundant casting too.